src/pages/experiments/[experimentId]/data-processing/configure-embedding/components/EmbeddingConfig.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import {
+  Collapse, InputNumber, Form, Select, Typography, Tooltip,
+} from 'antd';
+
+import { QuestionCircleOutlined } from '@ant-design/icons';
+
+const { Option } = Select;
+const { Panel } = Collapse;
+const { Text } = Typography;
+
+const MIN_DIST_TEXT = 'This controls how tightly the embedding is allowed compress points together. Larger values ensure embedded points are more evenly distributed, while smaller values allow the algorithm to optimise more accurately with regard to local structure. Expected range: 0.001 to 0.5. Default is 0.1.';
+
+const EmbeddingConfig = () => {
+  const [embeddingMethod, setEmbeddingMethod] = useState('umap');
+
+  const UMAPSettings = (
+    <>
+      <Form.Item>
+        <Text strong>Settings for UMAP:</Text>
+      </Form.Item>
+      <Form.Item label={(
+        <span>
+          Minimum distance&nbsp;
+          <Tooltip title={MIN_DIST_TEXT}>
+            <QuestionCircleOutlined />
+          </Tooltip>
+        </span>
+      )}
+      >
+        <InputNumber defaultValue={0.01} step={0.1} />
+      </Form.Item>
+      <Form.Item label='Distance metric:'>
+        <Select defaultValue='euclidean'>
+          <Option value='euclidean'>Euclidean</Option>
+          <Option value='cosine' disabled>Cosine</Option>
```

Comment:
maybe it is worth adding a tooltip with a description in here too, just like you did for min distance

Reply:
I don't think it's necessary, cosine and euclidean distances are not bioinformatics-specific things, they are just general ways to calculate distances between points (vectors), so you'd presumably know about them as a scientist. Vicky didn't add an explanation for it and I don't think there's much to be learned from one if you provide one -- you'd have to explain what each of them does anyways.

Reply:
sounds good then